libvirt/tests/src/virtual_network/attach_detach_device/attach_detach_ovs_interface.py

```
@@ -0,0 +1,155 @@
+    3. Attach OVS bridge interface by attach-device
+    4. Verify tap device is connected to OVS bridge
+    5. (Optional) Configure guest IP and ping bridge from guest
+    6. Detach interface by detach-device
+    7. Verify interface removed from live XML
+    8. Repeat steps 3-7 for requested number of iterations
+    """
+    vm_name = params.get('main_vm')
+    vm = env.get_vm(vm_name)
+    vmxml_backup = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
+
+    iface_attrs = eval(params.get('iface_attrs', '{}'))
```

Ruff (0.15.20)

[error] Use of possibly insecure function; consider using `ast.literal_eval`

(S307)